### PR TITLE
Feature/mip gnomad

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

New vcfanno toml files for gnomad r3.1

**How to prepare for test**:
- [ ] `ssh` to ...
- [ ] Install on stage:
`bash servers/resources/SERVER.scilifelab.se/update-[THIS_TOOL]-stage.sh [THIS-BRANCH-NAME]`

### How to test:
-

### Expected outcome:
- [ ]

### Review:
- [ ] Code approved by
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by jemten** [comment]
Hold off with this one for a little while. I need to confirm a few things

**Comment by jemten** [comment]
Should work now

File: rare-disease/annotation/grch37_vcfanno_config_-v1.16-.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+file="grch37_loqusdb_snv_indel_variants_export-20210416-.vcf.gz"
+fields = ["Frq", "Obs", "Hom"]
+ops=["self", "self", "self"]
+names=["Frq", "Obs", "Hom"]
+
+[[annotation]]
+file="grch37_gnomad_genomes_mt_-r3.1-.vcf.gz"
+fields = ["AF_hom", "AF_het", "mitotip_score", "mitotip_trna_prediction"]
+ops=["self", "self", "self", "self"]
+names=["MTAF_hom", "MTAF_het", "mitotip_score", "mitotip_trna_prediction"]
```
Comment (henrikstranneheim) [inline rare-disease/annotation/grch37_vcfanno_config_-v1.16-.toml:30]: In the future we should really try to standardise the name for these ones. Together with the Chiara and Daniel and Lund

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**